diff -uNr opt.orig/local/share/macports/Tcl/port1.0/portconfigure.tcl opt/local/share/macports/Tcl/port1.0/portconfigure.tcl
--- opt.orig/local/share/macports/Tcl/port1.0/portconfigure.tcl	2009-10-07 00:55:12.000000000 +0300
+++ opt/local/share/macports/Tcl/port1.0/portconfigure.tcl	2009-10-07 00:59:26.000000000 +0300
@@ -133,7 +133,7 @@
 default configure.cppflags  {-I${prefix}/include}
 default configure.cxxflags  {${configure.optflags}}
 default configure.objcflags {${configure.optflags}}
-default configure.ldflags   {-L${prefix}/lib}
+default configure.ldflags   {"-L${prefix}/lib -Xlinker -headerpad_max_install_names"}
 default configure.libs      {}
 default configure.fflags    {${configure.optflags}}
 default configure.f90flags  {${configure.optflags}}
diff -uNr opt.orig/local/var/macports/sources/rsync.macports.org/release/ports/x11/gtk2/Portfile opt/local/var/macports/sources/rsync.macports.org/release/ports/x11/gtk2/Portfile
--- opt.orig/local/var/macports/sources/rsync.macports.org/release/ports/x11/gtk2/Portfile	2009-10-07 00:56:21.000000000 +0300
+++ opt/local/var/macports/sources/rsync.macports.org/release/ports/x11/gtk2/Portfile	2009-10-07 01:00:41.000000000 +0300
@@ -85,6 +85,7 @@
 depends_run     port:shared-mime-info
 
 configure.args  --enable-static \
+                --with-included-loaders \
                 --disable-glibtest
 
 if { [variant_isset universal] } {
diff -uNr opt.orig/local/var/macports/sources/rsync.macports.org/release/ports/x11/pango/Portfile opt/local/var/macports/sources/rsync.macports.org/release/ports/x11/pango/Portfile
--- opt.orig/local/var/macports/sources/rsync.macports.org/release/ports/x11/pango/Portfile	2009-10-07 00:56:22.000000000 +0300
+++ opt/local/var/macports/sources/rsync.macports.org/release/ports/x11/pango/Portfile	2009-10-07 01:01:53.000000000 +0300
@@ -118,6 +118,7 @@
 
 configure.args \
     --enable-static \
+    --with-included-modules \
     --x-include=${prefix}/include --x-lib=${prefix}/lib
 
 variant no_x11 {
